### PR TITLE
gcs: esc calibration: send no pulses for 1.25s first

### LINKED ISSUE #1170: ESC calibration safety improvement?
Can we send no PWM for a bit to disarm the ESC, in case the user didn't listen and still has vbat connected?

### PR DESCRIPTION
**Pull request by mlyle**
Fixes #1170.

If the user doesn't listen and detach the battery and props, this gives
the ESC a chance to disarm by having no pulses.  No guarantees, etc...
Just slightly safer.

### CODE REVIEW

File: ground/gcs/src/plugins/config/configoutputwidget.cpp [hunk 1]
```diff
@@ -366,6 +366,29 @@ void ConfigOutputWidget::startESCCalibration()
...
+
+    QEventLoop loop;
+
+    QTimer timeout;
+    timeout.setSingleShot(true);
+    connect(&timeout, SIGNAL(timeout()), &loop, SLOT(quit()));
+    timeout.start(1250);        // 1.25 seconds
+    loop.exec();
+
+    actuatorCommand->setData(actuatorCommandFields);
```
Comment (tracernz) [inline ground/gcs/src/plugins/config/configoutputwidget.cpp]: Don't you want to do this before loop.exec() so the output is disabled during that 1.25 s?

**Review by tracernz — CHANGES_REQUESTED** [review]

File: ground/gcs/src/plugins/config/configoutputwidget.cpp (hunk 1 above)
Comment (mlyle) [reply ground/gcs/src/plugins/config/configoutputwidget.cpp]: LOL that might help huh :)

**Review by tracernz — APPROVED** [review]

**Comment by tracernz** [comment]
Looks good now, I'm confident it will work, but have you tested at all? :stuck_out_tongue:

**Comment by mlyle** [comment]
i have not yet, no :(

**Comment by tracernz** [comment]
I really need to double-check this and merge. :D

**Comment by tracernz** [comment]
Works great. :+1:

**Pull request merged**